### DISCUSSION TITLE
My BlueSCSI v2 can't mount more than one harddrive? (Powerbook 170)

### DISCUSSION THREAD

**Discussion by Missedisland** [BlueSCSI/BlueSCSI-v2#116] [Help]
Hey all, I've just upgraded my Powerbook 170 from a powerbook BlueSCSI v1 to a powerbook BlueSCSI v2 (with wifi pico). Somehow I have a strange problem where it now only boots with one of my drives (HD00 with OS 7.1) instead of multiple drives that are on the SD card. 

I've tried different SD cards, re-formatting, different types of hard drives (like new empty ones with Disk Jockey, pre-built ones and even the BlueSCSIToolkit.hda), but nothing loads besides the boot drive. While with the v1 BlueSCSI everything is fine. All drives have the same SCSI drivers and I'm on the latest firmware. I've also tried changing the SCSI termination jumper from ON to OFF, but then it doesn't boot anything at all (floppy with question mark). Any ideas what could be causing this? :) Thanks!

Here's an example of one of my log files which says the drives are loading fine, but they simply don't appear in the OS:

Platform: BlueSCSI Pico
FW Version: 2023.11.16-release Nov 17 2023 03:26:27
I2C Supported
Flash chip size: 2048 kB
 
=== SD Card Info ===
SD card detected, exFAT volume size: 122208 MB
SD Name: JD2S5, MID: 0x1B, OID: 0x53 0x4D
 
=== Global Config ===
Config file bluescsi.ini not found, using defaults
Active configuration:
 
=== Finding images in / ===
== Opening /HD00_512 Ultimate System 7.1.hda for ID: 0 LUN: 0
---- Image ready
== Opening /NE5.hda.txt for ID: 5 LUN: 0
---- Configuring as network based on image name
---- Image ready
== Opening /HD20_512 2048MB.hda for ID: 2 LUN: 0
---- Image ready

=== ROM Drive ===
Platform supports ROM drive up to 1692 kB
---- ROM drive image not detected
 
=== Configured SCSI Devices ===
* ID: 0, BlockSize: 512, Type: Fixed, Quirks: Apple, Size: 2048064kB
* ID: 2, BlockSize: 512, Type: Fixed, Quirks: Apple, Size: 2097216kB
* ID: 5, Type: Network, Quirks: Apple
 
=== Network Initialization ===
Wi-Fi MAC: 00:80:19:10:0F:49
Connecting to Wi-Fi SSID "" with no authentication
 
Initialization complete!
INFO: Pico Voltage: 3.239V.

**Comment by erichelgeson** [comment]
This can happen when there are different SCSI drivers on each device. Disk Jockey can fix this for you. Could you drag the "Ultimate System 7.1" to DJ and make sure it's using the Apple SCSI drivers? 

Also can you use a SCSI utility like SCSI Probe or similar to ensure you can see the other drives, they're just not mounted?

**Reply by Missedisland** [reply]
Thanks Eric. 

The behaviour is really strange. All drives have the same recommended Apple SCSI driver in DJ. the HD00 drive with the OS on it always works. The others don't load but when I open SCSI probe on the Powerbook I can see and mount all drives just fine. As soon as I reboot the laptop they're gone again.

At one point I added a new empty drive onto the SD card from DJ and System 7.1 gave me a prompt at boot that there was a new drive detected and it needed to initialise/wipe the drive to use it. I hit yes, because it was empty anyway. All of a sudden all drives initialized just fine and are now also there when booting up the laptop :) However, this only works everytime when the laptop was completely shutdown and then booted. When I hit restart they drives dissapear. Everything works fine when I use the BlueSCSI v1.

**Reply by erichelgeson** [reply]
Could you please try SCSI Probe or similar to see what drives are actually there? The process of "mounting" a drive is on the MacOS side, BlueSCSI just presents the disk and the Mac OS is what actually mounts them. I just want to rule out the BlueSCSI as having any issues. Number of drives doesn't really matter.

I'd also suggest you go through each step in the [Trouble Shooting guide](https://github.com/BlueSCSI/BlueSCSI-v2/wiki/Troubleshooting) in the wiki - can resolve most weird issues.

**Reply by CS-Mac2024** [reply]
I have the same issue with Macintosh Classic. And it is probably problem with Mac OS. When i boot my BlueSCSI V2 with system 6 it mounts all disk images on BlueSCSI and floppy disk from real floppy disk drive. When i boot with system 7 it mounts only "Macintosh HD". It won't even read the floppy drive. With disk images on BlueSCSI helped installing SCSIProbe to the system folder on my Macintosh. In SCSIProbe i clicked to "options" and then i marked "Mount Volumes During Startup". But floppy drive still does not work...

Little trick for mounting disk images and floppy disk is open Chooser and then close it. But that doesn't solve the problem.

my english is terrible so i hope the text is understandable... :D
